pkg/gotohelm/transpiler_test.go

```diff
@@ -21,16 +21,23 @@ import (
 )
 
 type TestSpec struct {
-	Unsupported bool
-	Values      []map[string]any
+	Unsupported   bool
+	ValuesChanged bool
```

Comment:
Could you add a comment here describing what `ValuesChanged` does? Something like "ValuesChanged, when true, disable the default assertion that the global `.Values` field has not been mutated."

Reply:
Maybe I should rename the `AssertForValuesChanges`?

Reply:
I generally like terse names with good documentation rather than long names :P. How about `NoAssertValuesMutation`? That works with the zero value but double negatives can be a bit confusing.